src/screensComponents/modal/index.js

```diff
@@ -39,18 +39,28 @@ export default class Modal extends BaseComponent {
   renderTouchableOverlay() {
     const {overlayBackgroundColor, onBackgroundPress, accessibilityLabel = 'Dismiss'} = this.props;
     if (_.isFunction(onBackgroundPress) || !!overlayBackgroundColor) {
-      return (
-        <View
-          accessible
-          accessibilityLabel={accessibilityLabel}
-          accessibilityRole="button"
-          style={[styles.touchableOverlay, {backgroundColor: overlayBackgroundColor}]}
-        >
-          <TouchableWithoutFeedback onPress={onBackgroundPress}>
-            <View flex/>
-          </TouchableWithoutFeedback>
-        </View>
-      );
+      if (Constants.accessibility.isScreenReaderEnabled) {
```

Comment:
can we somehow unify those two separate renders? 
for instance you can pass the `useSafeArea` prop to view in order to render SafeAreaView.. 
maybe you can condition the other props.

Reply:
I've unified them, not sure it's more readable, let me know.